extensions/wikia/AdEngine/js/MonetizationServiceHelper.js

```diff
@@ -0,0 +1,45 @@
+define('ext.wikia.adEngine.monetizationsServiceHelper', [
+	'wikia.cache',
+	'wikia.loader',
+], function (cache, loader) {
+	'use strict';
+
+	var assets,
+		isLoaded = false,
+		cacheKey = 'monetization_ads',
+		ttl = 604800;
+
+	/**
+	 * @desc loads all assets for monetization ads
+	 */
+	function loadAssets() {
```

Comment:
I'd love to see this function in some wikia-general module. Since there are few places in our monolith where we have similar logic. But I'm not sure how complex it would be to extract it. It's probably a separate task.